src/index.html

```diff
@@ -6,9 +6,26 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#"><img src="images/logo.png" alt="MoYo"></a>
+      </div>
+      <nav class="meny">
+        <ul class="gadges">
+          <li data-qa="hover" class="is-active"><a href="#####">Apple</a></li>
```

Comment:
```suggestion
          <li>
            <a href="#" class="is-active">Apple</a>
          </li>
```

Reply:
The same code style for other elements.

Reply:
You have not to use the data-qa attribute here.
![image](https://github.com/mate-academy/layout_moyo-header/assets/108701488/e6bd5fd5-2713-441e-b4aa-c919200b1588)